### ISSUE TITLE
sync randomly get stuck

### ISSUE THREAD

**Issue by raffis** [gyselroth/balloon-client-desktop#29]
#### Description
**Is**:
Sync does randomly get stuck. In development mode the sync window keeps beeing opened. 
Last log output:

```
{"level":"info","message":"Starting Sync with remote cursor: undefined","timestamp":"2017-11-15T07:41:08.824Z"}
{"attributes":["parent","hash","version","size"],"level":"info","message":"API: GET request to /node/delta with params: ","timestamp":"2017-11-15T07:41:08.873Z"}
{"attributes":["parent","hash","version","size"],"cursor":"xxxxxxxxxx","level":"info","message":"API: GET request to /node/delta with params: ","timestamp":"2017-11-15T07:41:11.485Z"}
```
**no more action here, even after a minute**

**Should**:
Sync continues.

#### balloon desktop version
0.0.32-dev, but seen in 0.0.31 as well.

#### Your client operating system
Linux Mint 17.1

#### Filesystem (If known)
ext4

#### balloon server version (If known)
1.0.16

**Comment by raffis** [comment]
this related to #39

**Comment by raffis** [comment]
fixed with 0.0.34

**Comment by raffis** [comment]
see #78
